.github/workflows/actions.yml

```diff
@@ -0,0 +1,63 @@
+# 59 23 * * *
+
+name: Update natural gas data
+
+on:
+  # Schedule to run on the 1st day of each month
+  schedule:
+    - cron: '59 23 * * *'
+
+  # Run on push to master branch
+  push:
```

Comment:
@gradedSystem so what happens if I run the workflow multiple times per day? Wouldn't it create a duplicate data?

Reply:
@gradedSystem ignore this comment please, I guess we just regenerate entire data file so no need to worry about duplicates.